tb/cpu_uvm/tests/test_random_uvm.py

```
@@ -0,0 +1,149 @@
+    num_seeds = 100
+    num_instructions = 100
+
+    for seed_idx in range(num_seeds):
+        seed = 1000 + seed_idx  # Deterministic seed sequence
+        dut._log.info(f"\n{'='*60}")
+        dut._log.info(f"Seed {seed_idx + 1}/{num_seeds}: {seed}")
+        dut._log.info(f"{'='*60}\n")
+
+        # Create and run test
+        test = RandomTest(f"random_test_seed{seed}", None, dut,
+                         num_instructions=num_instructions, seed=seed)
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)